secretcache/cacheVersion.go

```diff
@@ -59,7 +59,7 @@ func (cv *cacheVersion) refresh() {
 		cv.err = err
 		delay := exceptionRetryDelayBase * math.Pow(exceptionRetryGrowthFactor, float64(cv.errorCount))
 		delay = math.Min(delay, exceptionRetryDelayMax)
-		delayDuration := time.Nanosecond * time.Duration(delay)
+		delayDuration := time.Millisecond * time.Duration(delay)
```

Comment:
From cacheObject.go:
```
exceptionRetryDelayBase    = 1
exceptionRetryGrowthFactor = 2
exceptionRetryDelayMax     = 3600
```
I may be misunderstanding this logic, but it looks like the maximum backoff will be 3600 ms or 3.6 seconds. When I see 3600, I assumed that was in seconds so shouldn't this be time.Second?